packages/tools/examples/stackToVolumeWithSegmentations/index.ts

```diff
@@ -0,0 +1,430 @@
+let volumeToolGroup;
+
+const orientationOptions = {
+  axial: 'axial',
+  sagittal: 'sagittal',
+  coronal: 'coronal',
+};
+
+addButtonToToolbar({
+  title: 'Switch StackViewport to VolumeViewport, and vice versa',
+  onClick: async () => {
+    // Get the rendering engine
+    const renderingEngine = getRenderingEngine(renderingEngineId);
+
+    const viewport = renderingEngine.getViewport(viewportId);
+
+    let newViewport;
+    if (viewport.type === ViewportType.STACK) {
+      segmentation.state.removeSegmentationRepresentations(volumeToolGroupId);
+
+      newViewport = await csUtils.convertStackToVolumeViewport({
+        viewport: viewport as Types.IStackViewport,
+        options: {
+          background: <Types.Point3>[0, 0.4, 0],
+          volumeId: `cornerstoneStreamingImageVolume:myVolume`,
+        },
+      });
+
+      if (volumeToolGroup) {
+        volumeToolGroup.addViewport(newViewport.id, renderingEngineId);
+      }
+
+      segmentation.convertStackToVolumeSegmentation({
+        segmentationId: stackSegmentationId,
+        options: {
+          toolGroupId: volumeToolGroupId,
+          volumeId: `cornerstoneStreamingImageVolume:segMyVolume`,
+          newSegmentationId: volumeSegmentationId,
+        },
+      });
+
+      if (volumeToolGroup) {
+        volumeToolGroup.addViewport(newViewport.id, renderingEngineId);
+
+        volumeToolGroup.setToolDisabled('CircularBrush');
+        volumeToolGroup.setToolActive('SphereBrush', {
+          bindings: [
+            {
+              mouseButton: MouseBindings.Primary, // Left Click
+            },
+          ],
+        });
+      }
+    } else {
+      segmentation.state.removeSegmentationRepresentations(stackToolGroupId);
+
+      segmentation.convertVolumeToStackSegmentation({
+        segmentationId: volumeSegmentationId,
+        options: {
+          toolGroupId: stackToolGroupId,
+          newSegmentationId: stackSegmentationId,
+        },
+      });
+
+      newViewport = await csUtils.convertVolumeToStackViewport({
+        viewport: viewport as Types.IVolumeViewport,
+        options: {
+          background: <Types.Point3>[0.4, 0, 0.4],
+        },
+      });
+
+      // Set the tool group on the viewport
+      if (stackToolGroup) {
+        stackToolGroup.addViewport(newViewport.id, renderingEngineId);
+
+        stackToolGroup.setToolDisabled('SphereBrush');
+        stackToolGroup.setToolActive('CircularBrush', {
+          bindings: [
+            {
+              mouseButton: MouseBindings.Primary, // Left Click
+            },
+          ],
+        });
+      }
+    }
+
+    addOrientationDropdownIfVolumeViewport();
+  },
+});
+
+function addOrientationDropdownIfVolumeViewport() {
+  const renderingEngine = getRenderingEngine(renderingEngineId);
+  const viewport = renderingEngine.getViewport(viewportId);
+
+  // Check if the current viewport is a VolumeViewport
+  if (viewport.type === ViewportType.ORTHOGRAPHIC) {
+    addDropdownToToolbar({
```

Comment:
There is no corresponding remove when you drop it back to stack, so you get more and more dropdowns as you add.

Reply:
There is down below

```js
const orientationDropdown = document.getElementById('orientationDropdown');
    if (orientationDropdown) {
      orientationDropdown.remove();
    }
```